torch/optim/adam.py

```diff
@@ -105,9 +105,12 @@ class Adam(Optimizer):
         capturable (bool, optional): whether this instance is safe to capture in a CUDA graph.
             Passing True can impair ungraphed performance, so if you don't intend to
             graph capture this instance, leave it False (default: False)
-        fused (bool, optional): whether fused implementation of optimizer is used.
+        fused (bool, optional): whether the fused implementation (CUDA only) is used.
```

Comment:
nit: it is not really cuda-only? You could use it for CPU. And other device like xla/mps/xpu might benefit from it as well.
Or you prefer to mention this for now and update it when we get more implementations?

Reply:
Thanks for the review! I thought it was CUDA only since _fused_adam_ only has a CUDA dispatch, but yea we can update this when there are more implementations.
https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/native_functions.yaml#L14471-L14476